packages/block-editor/src/components/block-list/style.scss

```diff
@@ -945,60 +926,24 @@
 
 
 /**
- * Hover label
+ * Block Label for Navigation/Selection Mode
  */
 
 .block-editor-block-list__breadcrumb {
-	position: absolute;
-	line-height: 1;
-	z-index: z-index(".block-editor-block-list__breadcrumb");
-
-	// Position in the top left of the border.
-	left: -$block-padding - $block-left-border-width;
-	top: (($block-padding * -2) - $block-left-border-width);
-
-	.components-toolbar {
-		border: none;
-		line-height: 1;
-		font-family: $default-font;
-		font-size: 11px;
-		padding: 4px 4px;
-		background: $light-gray-500;
-		color: $dark-gray-900;
-		transition: box-shadow 0.1s linear;
-		@include reduce-motion("transition");
-
-		.components-button {
-			font-size: inherit;
-			line-height: inherit;
-			padding: 0;
-		}
-
-		.is-dark-theme & {
-			background: $dark-gray-600;
-			color: $white;
-		}
-	}
-
-	// Remove negative left breadcrumb position for left aligned blocks.
-	[data-align="left"] & {
-		left: 0;
-	}
-
-	// Right-align the breadcrumb for right-aligned blocks.
-	[data-align="right"] & {
-		left: auto;
-		right: 0;
-	}
+	display: none;
```

Comment:
Now, this is only rendered in navigation mode right? we could simplify the block.js code to do this and avoid doing it in CSS.  Fine to do it later though.

Reply:
Yes, this would be shown only in navigation/select mode.